app/org/sagebionetworks/bridge/dynamodb/DynamoSurveyResponse.java

```diff
@@ -202,49 +202,19 @@ public int hashCode() {
     public boolean equals(Object obj) {
         if (this == obj)
             return true;
-        if (obj == null)
-            return false;
-        if (getClass() != obj.getClass())
+        if (obj == null || getClass() != obj.getClass())
             return false;
         DynamoSurveyResponse other = (DynamoSurveyResponse) obj;
-        if (answers == null) {
-            if (other.answers != null)
-                return false;
-        } else if (!answers.equals(other.answers))
-            return false;
-        if (completedOn != other.completedOn)
-            return false;
-        if (identifier == null) {
-            if (other.identifier != null)
-                return false;
-        } else if (!identifier.equals(other.identifier))
-            return false;
-        if (healthCode == null) {
-            if (other.healthCode != null)
-                return false;
-        } else if (!healthCode.equals(other.healthCode))
-            return false;
-        if (startedOn != other.startedOn)
-            return false;
-        if (surveyGuid == null) {
-            if (other.surveyGuid != null)
-                return false;
-        } else if (!surveyGuid.equals(other.surveyGuid))
-            return false;
-        if (surveyCreatedOn != other.surveyCreatedOn)
-            return false;
-        if (version == null) {
-            if (other.version != null)
-                return false;
-        } else if (!version.equals(other.version))
-            return false;
-        return true;
+        return (Objects.equals(healthCode, other.healthCode) && Objects.equals(identifier, other.identifier) && 
+            Objects.equals(surveyKey, other.surveyKey) && Objects.equals(surveyGuid, other.surveyGuid) && 
+            Objects.equals(surveyCreatedOn, other.surveyCreatedOn) && Objects.equals(survey, other.survey) && 
+            Objects.equals(startedOn, other.startedOn) && Objects.equals(completedOn, other.completedOn) && 
+            Objects.equals(answers, other.answers));
     }
```

Comment:
hashCode and equals are both missing version. Do we care about that field? Should we add an EqualsVerifier test for these?